### ISSUE TITLE
For booru owner it shows "posting anonymously" when logged in.

### ISSUE THREAD

**Issue by Seedmanc** [Seedmanc/Booru-mass-uploader#23]
Uploads correctly though, caused by different cookies for owners.

**Comment by Seedmanc** [comment]
Please report if you have experienced the same issue, I'm not sure if this really happens or it was a coincidence of sorts.

**Comment by ProximaNova** [comment]
I had the same problem. With at least one booru which runs Danbooru software, it says "You are posting anonymously to a danbooru site." even though I just logged in and uploaded multiple files there, resulting in the log message "[20:39] Finished uploading; 18 uploaded ok + 3 failed = 21 images total."

**Comment by Seedmanc** [comment]
Hard for me to fix since I only work with a booru.org website, I don't feel like spamming existing danbooru sites with test uploads or bother with launching one myself.

**Comment by ProximaNova** [comment]
OK. It's not a big problem I guess. Also, I think I made a pull request which might have meant that you should update your software based on my software. That was a mistake; I was trying to update my fork of your software but didn't really know how to use this github or "git" system to do that.

**Comment by ProximaNova** [comment]
"I don't feel like spamming existing danbooru sites with test uploads"

I could send you some test files which would not be spam for a website I know of, but I think I read that you live in Russia. Is that true?

**Comment by Seedmanc** [comment]
Yes and I'm not too eager about the active development of this project anymore, best not to break what works now. I don't know if the current issue is a major inconvenience.
